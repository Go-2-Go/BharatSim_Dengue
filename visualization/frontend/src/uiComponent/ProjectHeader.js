import React from 'react';
import { Box, Typography } from '@material-ui/core';

import { makeStyles } from '@material-ui/core/styles';
import { ChildrenPropTypes } from '../commanPropTypes';

const useProjectLayoutStyle = makeStyles((theme) => {
  return {
    projectNameBar: {
      textTransform: 'capitalize',
      boxShadow: '0px 1px 1px rgba(78, 96, 176, 0.3)',
      height: theme.spacing(12),
      padding: theme.spacing(0, 8),
      display: 'flex',
      alignItems: 'center',
      justifyContent: 'space-between',
    },
  };
});

function ProjectHeader({ children }) {
  const classes = useProjectLayoutStyle();

  return (
    <Box className={classes.projectNameBar}>
      <Typography variant="h5">{children}</Typography>
    </Box>
  );
}

ProjectHeader.propTypes = {
  children: ChildrenPropTypes.isRequired,
};

export default ProjectHeader;
