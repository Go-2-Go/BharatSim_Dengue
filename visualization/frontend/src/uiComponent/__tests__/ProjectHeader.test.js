import React from 'react';
import { render } from '@testing-library/react';
import ProjectHeader from '../ProjectHeader';

describe('<ProjectHeader />', () => {
  it('should match snapshot', () => {
    const { container } = render(<ProjectHeader>Title</ProjectHeader>);

    expect(container).toMatchSnapshot();
  });
});
