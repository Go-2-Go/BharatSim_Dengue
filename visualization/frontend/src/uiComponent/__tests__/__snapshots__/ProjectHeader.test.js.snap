// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProjectHeader /> should match snapshot 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-2 makeStyles-projectNameBar-1"
  >
    <h5
      class="MuiTypography-root MuiTypography-h5"
    >
      Title
    </h5>
  </div>
</div>
`;
